### PR TITLE
Issue137 performance json

### PR DESCRIPTION
**Pull request by geoperez**

### CODE REVIEW

**Comment by geoperez** [comment]
Check benchmark in ticket #137

File: test/Unosquare.Swan.Test/JsonTest.cs [hunk 1]
```diff
@@ -315,23 +307,20 @@ public void ObjectWithArrayWithData_ReturnsObjectWithArrayWithDataDeserialized()
...
             Assert.AreEqual(2, data.Id);
             Assert.AreEqual(1, data.Properties.Length);
         }
 
         [Test]
         public void WithJsonProperty_ReturnsPropertiesDeserialized()
         {
-            var obj = new JsonPropertySample() {Data = "OK", IgnoredData = "OK"};
+            var obj = new JsonPropertySample {Data = "OK", IgnoredData = "OK"};
             var data = Json.Serialize(obj);
```
Comment (Alexey-Unosquare) [inline test/Unosquare.Swan.Test/JsonTest.cs]: Are you testing Deserialize or Serialize method here?

Comment (geoperez) [reply test/Unosquare.Swan.Test/JsonTest.cs]: I will rewrite some tests on another branch.

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "express": "^4.16.3",
     "ldapjs": "^1.0.2",
-    "smtp-server": "^3.4.5"
+    "smtp-server": "^3.4.6"
```
Comment (Alexey-Unosquare) [inline package.json]: Keep only relevant changes in your PRs if possible

Comment (geoperez) [reply package.json]: Sure

File: src/Unosquare.Swan.Lite/Formatters/Json.Converter.cs [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+                if (targetInstance == null)
+                {
+                    // Try to create a default instance
+                    try
+                    {
+                        source.CreateTarget(_targetType, _includeNonPublic, ref _target);
+                    }
+                    catch
+                    {
+                        _target = _targetType.GetDefault();
```
Comment (Alexey-Unosquare) [inline src/Unosquare.Swan.Lite/Formatters/Json.Converter.cs]: what about moving this line in source.CreateTarget? and use try catch there?

File: src/Unosquare.Swan.Lite/Formatters/Json.Converter.cs [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+                        return;
+                    }
+                }
+                else
+                {
+                    _target = targetInstance;
+                }
```
Comment (Alexey-Unosquare) [inline src/Unosquare.Swan.Lite/Formatters/Json.Converter.cs]: return is missing?

Comment (geoperez) [reply src/Unosquare.Swan.Lite/Formatters/Json.Converter.cs]: Nop, the target is required in the following lines.

File: src/Unosquare.Swan.Lite/Formatters/Json.Converter.cs (hunk 3 above)
Comment (geoperez) [reply src/Unosquare.Swan.Lite/Formatters/Json.Converter.cs]: I think I can do it.

Comment (geoperez) [reply src/Unosquare.Swan.Lite/Formatters/Json.Converter.cs]: That is wrong. If the instance can not be created, the default value is returned and the process should exit.

**Pull request merged**